pkg/templates/aws/resources/ecs_task_definition.yaml

```diff
@@ -43,14 +56,20 @@ properties:
             type: string
             description: The value of the environment variable
         description: The environment variables to pass to a container
+      Essential:
+        type: bool
+        default_value: true
+        description: If the essential parameter of a container is marked as true, and
+          that container fails or stops for any reason, all other containers that
+          are part of the task are stopped
       Image:
         type: resource(aws:ecr_image)
         required: true
         operational_rule:
           step:
             direction: downstream
             resources:
-              - aws:ecr_image:{{ .Self.Name }}-{{ fieldValue (printf "%s.Name" (pathAncestor .Path 1)) .Self}}
+              - aws:ecr_image:{{ .Self.Name }}-{{ if pathAncestorExists .Path 1}}{{ if hasField (printf "%s.Name" (pathAncestor .Path 1)) .Self}}{{ fieldValue (printf "%s.Name" (pathAncestor .Path 1)) .Self}}{{else}}image{{end}}{{else}}image{{end}}
```

Comment:
I worry about using `fieldValue` for things that aren't ResourceIds since it won't work if it is imported. Not a concern for now, but just something we need to be aware of.